it/EditSetISpec.scala

```diff
@@ -0,0 +1,1099 @@
+import controllers.EditSetControllerSpec.ExpectedEditSetSummaryRow
+import play.api.http.Status.OK
+import play.api.libs.ws.{ DefaultWSCookie, WSCookie, WSResponse }
+import play.api.test.Helpers.{ await, defaultAwaitTimeout }
+import support.CommonMatchers._
+
+class EditSetISpec extends BaseISpec {
+
+  "when viewed, for a specified edit set ID" should {
```

Comment:
Could we use `must` rather than `should` please.

Reply:
Sure. I'll change them.

Reply:
I'll change them in `LoginISpec`, as well.

Reply:
Changed.

Reply:
Thanks